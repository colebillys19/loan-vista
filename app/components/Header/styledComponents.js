import styled from 'styled-components';

export const HeaderWrapper = styled.header`
  background-color: #7a6cd2;
  height: 8rem;
`;
