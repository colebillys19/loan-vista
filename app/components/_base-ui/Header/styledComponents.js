import styled from 'styled-components';

import { backgroundColor, appColorA, appColorB } from 'styleConstants';

export const HeaderContainer = styled.header`
  background-color: ${appColorA};
  font-size: 3rem;
  height: 8rem;
  letter-spacing: 0.5rem;
  min-width: 122rem;
  padding: 2.4rem 0 0 5rem;
  & span:first-of-type {
    color: ${backgroundColor};
  }
  & span:last-of-type {
    color: ${appColorB};
  }
`;
