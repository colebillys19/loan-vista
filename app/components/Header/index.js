/**
 * Header
 * @description ...
 */

import React from 'react';

import { HeaderWrapper, SwatchA, SwatchB } from './styledComponents';

const Header = () => (
  <HeaderWrapper>
    <SwatchA />
    <SwatchB />
  </HeaderWrapper>
);

export default Header;
