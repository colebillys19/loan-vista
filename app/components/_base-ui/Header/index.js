/**
 * Header
 * @description ...
 */

import React from 'react';

import { HeaderContainer } from './styledComponents';

const Header = () => (
  <HeaderContainer>
    <span>loan</span>
    <span>Vista</span>
  </HeaderContainer>
);

export default Header;
