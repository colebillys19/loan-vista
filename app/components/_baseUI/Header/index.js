/**
 * Header
 * @description ...
 */

import React from 'react';

import { HeaderContainer, SwatchA, SwatchB } from './styledComponents';

const Header = () => (
  <HeaderContainer>
    <SwatchA />
    <SwatchB />
  </HeaderContainer>
);

export default Header;
