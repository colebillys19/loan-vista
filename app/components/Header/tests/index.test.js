import React from 'react';
import { render } from 'react-testing-library';

import Header from '../index';

describe('<Header />', () => {
  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    render(<Header />);
    expect(spy).not.toHaveBeenCalled();
  });

  it('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(<Header />);
    expect(firstChild).toMatchSnapshot();
  });
});
