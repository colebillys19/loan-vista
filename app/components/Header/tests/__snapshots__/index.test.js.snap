// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Should render and match the snapshot 1`] = `
<header
  class="styledComponents__HeaderWrapper-sc-1jp3vlc-0 cOVGpl"
/>
`;
